graylog2-web-interface/src/components/navigation/Navigation.tsx

```diff
@@ -181,6 +182,11 @@ const Navigation = React.memo(({ pathname }: Props) => {
             {pluginItems.map(({ key, component: Item }) => <Item key={key} />)}
           </InactiveNavItem>
 
+          <LinkContainer relativeActive to={Routes.WELCOME}>
+            <NavItem id="welcome-nav-link" aria-label="Welcome">
+              <Icon size="lg" fixedWidth title="Welcome" name="table-columns" />
```

Comment:
What do you think about suing the home icon instead and placing it on the left side of the profile icon?
https://fontawesome.com/v5/icons/home?s=solid&f=classic

Reply:
@maxiadlovskii can you please take a look?

Reply:
@linuspahl This Icon was my first thought. But the welcome page might not be a home page for some users if they set up some custom home page.

Reply:
Even when I set a different start page, I still see the welcome page when navigating to `/welcome` by clicking on the icon. Since we always show the welcome page in this case I prefer the home icon.

What do you think about my suggestion regarding the position? I would move the icon next to the profile icon. This way we have internal links on the right side and everything else on the left side.